tests/auxtel/test_cam_take_image.py

```diff
@@ -161,8 +161,8 @@ async def test_take_images(self):
         async with self.make_script():
             config = await self.configure_script(
                 nimages=1,
-                exp_times=0,
-                image_type="BIAS",
+                exp_times=1.0,
```

Comment:
A general comment about these tests. Many of the scripts are not completely trivial *in this package* since all the work is in ts_telescope_control. Would it make any sense to move all these tests into that package and only test the very minimum here, e.g. that the script can be run?

Reply:
ah yes.. these where pre-existing tests... I was cross to remove them but indeed, it is duplicated since it is using the `LATISS` class...

Reply:
I like tests, but I hate duplication and the maintenance headaches. If the testing is done in the new package I think it would be best not to duplicate complex test code here.

I gently suggest either:
* Making the tests part of the ts_operation_manager library so you can run the same tests here and there without duplicating the test code.
*  Removing tests here that are duplicated in ts_operation_manager (but only when the run method is truly trivial, as it now is for so many scripts).